### PR TITLE
Double counts

### PR DESCRIPTION
**Pull request by bart-de-jonge**
Changed all int counter stuff to quarters (4 grid cells per count).
That means the input in some textfields is parsed to quarters when exiting focus or pressing enter, and converting these doubles to the right cells in the grid pane.

### CODE REVIEW

**Comment by martijnjanssen** [comment]
Looks good, can't really find anything weird.

**Comment by geenen124** [comment]
Maybe create a count utility class for string formatting and the cell counts? This way you don't have to redefine/change the counts in every class if/when this changes.

File: src/main/java/control/CountUtilities.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package control;
+
+/**
+ * Created by Bart on 10/05/2016.
+ */
+public class CountUtilities {
+
+    public static final int numberOfCellsPerCount = 4;
```
Comment (geenen124) [inline src/main/java/control/CountUtilities.java]: Static final variables should be all caps

**Comment by geenen124** [comment]
Looks good! I'll merge once the build passes

**Pull request merged**